service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -356,7 +356,7 @@ fn generate_body_parser(operation: &Operation, service: &Service<'_>) -> String
                 Some(ref s) => {
                     // if there's any required shape present the body payload parser will handle it
                     // This can't be converted to `s.is_empty()`. TODO: find out why.
-                    s.len() > 0
+                    !s.empty()
```

Comment:
I think this should be `is_empty()` instead of `empty()`. 😄

Reply:
Ok now it just work! I cannot believe that i didn't see this typo